### PR TITLE
feature/avatar-alert

### PR DESCRIPTION
**Pull request by surajmore98**
- created struture for component library.
- avatar component added.
avatar

- alert component added.
alert

### CODE REVIEW

**Review by ravikumar1002 — APPROVED** [review]

File: src/style/avatar.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    font-size: 2rem;
+    line-height: 3rem;
+}
+
+.avatar-small {
+    height: 25px;
+    width: 25px;
+    font-size: 1rem;
+    line-height: 1.5rem;
+}
```
Comment (romabulani) [inline src/style/avatar.css:34]: Add rems instead of px for height and width of avatars

Comment (surajmore98) [reply src/style/avatar.css:34]: @romabulani value will not change for avatar with screen size change therefor i have used px instead of rem.

Comment (surajmore98) [reply src/style/avatar.css:34]: @romabulani avatar size is fixed inrespective of screen size therefore used px instead of rem.

Comment (romabulani) [reply src/style/avatar.css:34]: Yes, as rems is based on font size, it will be good with responsiveness. But if you want to keep it fixed, then its fine.

**Review by romabulani — APPROVED** [review]

**Pull request merged**